README.md

```diff
@@ -761,9 +769,9 @@ v::consonant()->validate('xkcd'); //true
 
 See also:
 
-  * [v::alnum()](#valnum)  - a-z0-9, empty or whitespace only
-  * [v::digit()](#vdigit) - 0-9, empty or whitespace only
-  * [v::alpha()](#valpha)  - a-Z, empty or whitespace only
+  * [v::alnum()](#valnum)  - a-z0-9 or empty
+  * [v::digit()](#vdigit) - 0-9 or empty
+  * [v::alpha()](#valpha)  - a-z or empty
```

Comment:
We appreciate these changes, but you should sent a separated pull request for this. 

Each pull request must be one only subject, so it can easily reviewed and it increases the chance of merge; if this change on README.md was send separately it was already merged.